src/rest/run.py

```diff
@@ -2,18 +2,20 @@
 
 import logging
 import multiprocessing
-import waitress
-from src.rest.runner import Runner
+
+import uvicorn
```

Comment:
this is not in requirements.text...should it be?

Reply:
huh, I did put it in I thought o.0
Thx for the good eyes